kbase-extension/static/kbase/js/widgets/common/JSONView.js

```diff
@@ -0,0 +1,63 @@
+define(['jquery', 'widgets/common/RenderIn'], ($, RenderIn) => {
+    'use strict';
+
+    const { $TextIn } = RenderIn;
+
+    function $RenderJSONArray(data) {
+        const $rows = data.map((value, index) => {
+            return $('<tr>').append(
+                $('<th>').css('color', 'rgba(150, 150, 150, 1)').text(String(index)),
```

Comment:
can you move all the css declarations be moved out of these JS files and into the appropriate css file?

Reply:
Not sure it is worth the added complexity, but sure. Added css file and created some associated classes. 
I realize the Narrative has moved most css into scss - should this as well?